src/app/redux/UserSaga.js

```diff
@@ -635,9 +636,21 @@ function* uploadImage({
     };
     xhr.upload.onprogress = function(event) {
         if (event.lengthComputable) {
-            const percent = Math.round(event.loaded / event.total * 100);
+            const percent = Math.round(event.loadedx / event.total * 100);
             progress({ message: `Uploading ${percent}%` });
         }
     };
     xhr.send(formData);
 }
+
+function* hidePinnedPost(action) {
+    console.log('userSaga hidePinnedPost', action);
+    const closedPinnedPosts = yield select(state =>
+        state.user.get('closedPinnedPosts')
+    );
+    console.log('userSaga closedPinnedPosts', closedPinnedPosts);
```

Comment:
If you want to keep the console calls around, change them to `console.info` if you don't mind